tests/test_wps_generic_zonal_stats.py

```diff
@@ -109,15 +100,10 @@ def test_simple_geojson(self):
 
         assert_response_success(resp)
         out = get_output(resp.xml)
+        feature = json.loads(out['statistics'])['features'][0]
```

Comment:
This only checks the first feature, right? Why not all?

Reply:
It's the same logic as the original test. 
And it's just a smoke test, we're still not checking the actual values.